app/templates/events/view/tickets/index.hbs

```diff
@@ -4,7 +4,7 @@
   </div>
   <div class="ui {{if this.device.isMobile 'horizontal'}} three statistics">
     <div class="statistic"><div class="value">{{this.totalOrders}}</div><div class="label">{{t 'Orders'}}</div></div>
-    <div class="statistic"><div class="value">{{round this.totalAmount decimals=2}}</div><div class="label">{{t 'Sales'}}{{#if this.totalAmount}}({{currency-symbol this.model.eventDetail.paymentCurrency}}){{/if}}</div></div>
+    <div class="statistic"><div class="value">{{#if (eq this.totalAmount 0)}}{{t '0.00'}}{{else}}{{round this.totalAmount decimals=2}}{{/if}}</div><div class="label">{{t 'Sales'}}{{#if this.totalAmount}}({{currency-symbol this.model.eventDetail.paymentCurrency}}){{/if}}</div></div>
```

Comment:
Now, please make it work for 1.00, 2.00, 3.00, ... ∞

Reply:
@iamareebjamal is this correct? Or should I register a handlebar helper in the js file?
![2020-11-28](https://user-images.githubusercontent.com/61330148/100477024-2af57f80-310d-11eb-96a5-3a202edb4249.png)

Reply:
No, it's not correct. Check the round helper documentation

Reply:
If the number is not having digits after decimal point then round helper will not do anything. Then how should we add .00 ?

Reply:
Then create a helper which calls `toFixed(2)` on the number

Reply:
In the return statement, I added this check, so round helper is not required now. Is it okay?